frontend/src/reducers/users.js

```diff
@@ -23,6 +23,8 @@ export default function users(state={
 
   switch (type) {
     case constants.USER_GROUPS_SUCCESS:
+    Object.keys(groups).map(value =>
+      groups[value] = Object.assign({}, groups[value], {balance: groups[value].balance/= 100}))
```

Comment:
Xavier had a better idea of doing this in the last step in the currency component, so we'll fix it as part of that.